src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -0,0 +1,165 @@
+  isUserAllowedToImplement,
+  isUserAllowedToDelete,
+  changeRequest,
+  handleReviewOpen,
+  handleDeleteOpen,
+  users
+}: ChangeRequestActionMenuProps) => {
+  const { mutateAsync: requestCRReview } = useRequestCRReview(changeRequest.crId.toString());
+  const toast = useToast();
+  const currentUser = useCurrentUser();
+  const history = useHistory();
+  const [reviewerIds, setReviewerIds] = useState<number[]>([]);
+
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) {
+      toast.error('Must select at least one reviewer to request review from');
+      return;
+    }
+    try {
+      await requestCRReview({ userIds: reviewerIds });
+    } catch (e) {
+      if (e instanceof Error) {
+        toast.error(e.message);
+      }
+    }
+  };
+
+  const isRequestAllowed = changeRequest.submitter.userId === currentUser.userId;
+
+  const UnreviewedActionsDropdown = () => (
+    <div style={{ marginTop: '10px' }}>
+      <ActionsMenu
+        divider={true}
+        buttons={[
+          {
+            title: 'Review',
+            onClick: handleReviewOpen,
+            disabled: !isUserAllowedToReview,
+            icon: <EditIcon fontSize="small" />
+          },
+          {
+            title: 'Delete',
+            onClick: handleDeleteOpen,
+            disabled: !isUserAllowedToDelete,
+            icon: <DeleteIcon fontSize="small" />
+          }
+        ]}
+      />
+    </div>
+  );
+
+  const RequestReviewerDropdown = () => (
+    <>
+      <Autocomplete
+        isOptionEqualToValue={(option, value) => option.id === value.id}
+        limitTags={1}
+        disableCloseOnSelect
+        multiple
+        options={users.filter((user) => isLeadership(user.role)).map(userToAutocompleteOption)}
+        getOptionLabel={(option) => option.label}
+        onChange={(_, values) => setReviewerIds(values.map((value) => value.id))}
+        renderTags={() => null}
+        renderOption={(props, option, { selected }) => (
+          <li {...props}>
+            <Checkbox
+              icon={<CheckBoxOutlineBlankIcon />}
+              checkedIcon={<CheckBoxIcon />}
+              style={{ marginRight: 8 }}
+              checked={selected}
+            />
+            {option.label}
+          </li>
+        )}
+        renderInput={(params) => (
+          <TextField {...params} variant="standard" placeholder={`${reviewerIds.length} Reviewers Selected`} />
+        )}
+      />
+
+      <Box display="flex" flexDirection="row" gap="10px" justifyContent="right">
+        <NERButton
+          sx={{ mt: '10px', float: 'right' }}
+          variant="contained"
+          disabled={!isRequestAllowed}
+          onClick={handleRequestReviewerClick}
+        >
+          Request Review
+        </NERButton>
+        <UnreviewedActionsDropdown />
+      </Box>
+    </>
+  );
+
+  const RenderUnreviewedActionsDropdown = () =>
+    isRequestAllowed && changeRequest.status === ChangeRequestStatus.Open ? (
```

Comment:
I think you should just include this `&& changeRequest.status === ChangeRequestStatus.Open ` in the code that calculates isRequestAllowed, since that's essentially what you're saying here: you should only be allowed to request a reviewer if the CR is open. Also the request button should only show up if the CR is open the way its written rn anyway